src/mon/PGMap.cc

```diff
@@ -807,7 +807,9 @@ void PGMapDigest::dump_pool_stats_full(
           << pool_id;
     }
     float raw_used_rate = osd_map.pool_raw_used_rate(pool_id);
-    dump_object_stat_sum(tbl, f, stat, avail, raw_used_rate, verbose, pool);
+    bool legacy = use_legacy_per_pool_stats();
```

Comment:
you can call this inside dump_object_stat_sum and avoid passing the additional parameter...

Reply:
can't, it's static